utils/environment.sh

```diff
@@ -44,3 +44,38 @@ if [ $(${XRAY_VIVADO} -h |grep Vivado |cut -d\  -f 2) != "v2017.2" ] ; then
     export XRAY_DIR="/bad/vivado/version"
     return
 fi
+
+# Check if database files exist, if don't create empty ones
```

Comment:
This doesn't belong in environment.sh.  Maybe a call in ${XRAY_PART}/settings.sh?

Reply:
sure, we can move it there but we'd have to call it in every settings file (for that I'd pack this into another script and call the script)
ROI harness minitest introduces it's own settings file. The only common part between all of them is the environment.sh, that is why I put it there.
Shall I add the call every setting file in the project?

Reply:
enviroment.sh might be common, but it should have side affects.

Have environment.sh define a function is fine, just call it in each settings.sh

Reply:
sure, I'll do it that way then

Reply:
done